source/jsonnet/england-wales/ccs/blocks/individual/another_uk_address.jsonnet

```diff
@@ -1,14 +1,28 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
 local rules = import 'rules.libsonnet';
 
-local nonProxyTitle = 'Is there another UK address where you may have been included on a census questionnaire because you were a usual resident, or staying overnight there on Sunday 13 October 2019?';
+local nonProxyTitle = {
+  text: 'Is there another UK address where you may have been included on a census questionnaire because you were a usual resident, or staying overnight there on Sunday {census_date}?',
```

Comment:
My initial reaction was that the day should be handled by the `format_date` placeholder transform (the day can be output by adding `EEEE` to the `date_to_format` property). Unfortunately the Welsh translation via Babel is then incorrect for our usage. It translates "Sunday" as "Dydd Sul", whereas our current translations use "ddydd Sul" (ddydd is lowercase mutates due to the "on" before the day).

So, in summary, I think this usage is acceptable.